user-managerd.pro

```diff
@@ -15,7 +15,7 @@ DEFINES += QT_DEPRECATED_WARNINGS
 # You can also select to disable deprecated APIs only up to a certain version of Qt.
 #DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
 
-PKGCONFIG += libuser glib-2.0 sailfishaccesscontrol
+PKGCONFIG += libuser glib-2.0 sailfishaccesscontrol libsystemd-login mce-qt5
```

Comment:
Needs matching build requires. Also I wonder if that libsystemd-login should be libsystemd or such. They did deprecate some of those names.